install/Makefile

```diff
@@ -29,7 +38,90 @@ crd_install_helm3:
 	yq eval -i '.metadata.labels.component="kafkamirrormaker2.kafka.strimzi.io-crd"' ../helm-charts/helm3/strimzi-kafka-operator/crds/048-Crd-kafkamirrormaker2.yaml 
 	yq eval -i '.metadata.labels.component="kafkarebalances.kafka.strimzi.io-crd"' ../helm-charts/helm3/strimzi-kafka-operator/crds/049-Crd-kafkarebalance.yaml 
 
-crd_install: crd_install_helm3
+update_bundle:
+	# Copy Custom Resource Definitions (CRDs)
+	$(CP) $(CRD_DIR)/043-Crd-kafkatopic.yaml $(MANIFESTS)/kafkatopics.kafka.strimzi.io.crd.yaml
+	$(CP) $(CRD_DIR)/044-Crd-kafkauser.yaml $(MANIFESTS)/kafkausers.kafka.strimzi.io.crd.yaml
+	$(CP) $(CRD_DIR)/040-Crd-kafka.yaml $(MANIFESTS)/kafkas.kafka.strimzi.io.crd.yaml
+	$(CP) $(CRD_DIR)/041-Crd-kafkaconnect.yaml $(MANIFESTS)/kafkaconnects.kafka.strimzi.io.crd.yaml
+	$(CP) $(CRD_DIR)/042-Crd-kafkaconnects2i.yaml $(MANIFESTS)/kafkaconnects2is.kafka.strimzi.io.crd.yaml
+	$(CP) $(CRD_DIR)/045-Crd-kafkamirrormaker.yaml $(MANIFESTS)/kafkamirrormakers.kafka.strimzi.io.crd.yaml
+	$(CP) $(CRD_DIR)/046-Crd-kafkabridge.yaml $(MANIFESTS)/kafkabridges.kafka.strimzi.io.crd.yaml
+	$(CP) $(CRD_DIR)/047-Crd-kafkaconnector.yaml $(MANIFESTS)/kafkaconnectors.kafka.strimzi.io.crd.yaml
+	$(CP) $(CRD_DIR)/048-Crd-kafkamirrormaker2.yaml $(MANIFESTS)/kafkamirrormaker2s.kafka.strimzi.io.crd.yaml
+	$(CP) $(CRD_DIR)/049-Crd-kafkarebalance.yaml $(MANIFESTS)/kafkarebalances.kafka.strimzi.io.crd.yaml
+	$(CP) $(CRD_DIR)/030-ClusterRole-strimzi-kafka-broker.yaml $(MANIFESTS)/strimzikafkabroker.clusterrole.yaml
+	$(CP) $(CRD_DIR)/031-ClusterRole-strimzi-entity-operator.yaml $(MANIFESTS)/strimzientityoperator.clusterrole.yaml
+	$(CP) $(CRD_DIR)/032-ClusterRole-strimzi-topic-operator.yaml $(MANIFESTS)/strimzitopicoperator.clusterrole.yaml
+	$(CP) $(CRD_DIR)/033-ClusterRole-strimzi-kafka-client.yaml $(MANIFESTS)/strimzikafkaclient.clusterrole.yaml
+	$(CP) $(CRD_DIR)/050-ConfigMap-strimzi-cluster-operator.yaml $(MANIFESTS)/strimziclusteroperator.configmap.yaml
+
+	# Update exisiting Cluster Service Version (CSV) file
+	yq ea -i 'select(fi==0).spec.install.spec.deployments[0].spec = select(fi==1).spec | select(fi==0)' $(CSV_FILE) $(DEPLOYMENT_FILE)
+	yq ea -i 'select(fi==0).spec.install.spec.permissions[0].rules = select(fi==1).rules | select(fi==0)' $(CSV_FILE) $(CRD_DIR)/020-ClusterRole-strimzi-cluster-operator-role.yaml
+	yq ea -i 'select(fi==0).spec.install.spec.clusterPermissions = select(fi==1).rules | select(fi==0)' $(CSV_FILE) $(CRD_DIR)/021-ClusterRole-strimzi-cluster-operator-role.yaml
```

Comment:
You need to check that this command correctly creates the `spec.install.spec.clusterPermissions.rules` list. When I tried it, it copied the entries below the `- rules:` key but missed the `- rules:` itself.

Reply:
The `clusterPermissions` was also missing a `serviceAccountName` entry

Reply:
Actually this looks like an issue with your bundle template, see comments below.

Reply:
Thanks for the catch, just fixed it up!